pkgs/development/python-modules/thefuzz/default.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, python3
+, buildPythonPackage
+, fetchPypi
+, python-Levenshtein
+, pycodestyle
+, hypothesis
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "thefuzz";
+  version = "0.19.0";
+  format = "setuptools";
+
+  disabled = python3.pkgs.pythonOlder "3.7";
```

why not to be consistent and

```suggestion
  disabled = pythonOlder "3.7";
```

and add it in the inputs list?